cmd/access/node_builder/access_node_builder.go

```diff
@@ -1677,6 +1684,11 @@ func (builder *FlowAccessNodeBuilder) Build() (cmd.Node, error) {
 				return err
 			}
 
+			builder.BlockTransactions, err = stdmap.NewIdentifierMap(100)
```

Comment:
100 is a bit too small I feel. Let's increase this to `10000`, it's cheap to hold `10000` identifiers in memory, which is around `320KB` in total. (each identifier is 32 bytes)
```suggestion
			builder.BlockTransactions, err = stdmap.NewIdentifierMap(10000)
```